### PR TITLE
Add messages to indicate the assigned user and the locked by user; disable inputs if locked

### PR DESCRIPTION
**Pull request by thomasmunduchira**
* Show a message that indicates the assigned user of an application if an admin is looking at an application that is not assigned to them
* Show a message that indicates the admin who locked the user's application if applicable
* Inputs are now disabled if the application has been locked and the user is not an admin

### CODE REVIEW

File: applications-frontend/src/components/Applicant.vue [hunk 1]
```diff
@@ -114,6 +117,15 @@ export default {
     };
   },
   computed: {
+    isReadOnly: function() {
+      return this.isReadOnlyForUser();
+    },
+    isDisabled: function() {
+      return !this.application.id || this.isReadOnlyForUser();
+    },
+    isAssignedDifferent: function() {
```
Comment (klanmiko) [inline applications-frontend/src/components/Applicant.vue]: I would prefer if you moved the method body into these methods

File: applications-frontend/src/components/Applicant.vue [hunk 2]
```diff
@@ -114,6 +117,15 @@ export default {
     };
   },
   computed: {
+    isReadOnly: function() {
+      return this.isReadOnlyForUser();
+    },
+    isDisabled: function() {
+      return !this.application.id || this.isReadOnlyForUser();
```
Comment (klanmiko) [inline applications-frontend/src/components/Applicant.vue]:
```suggestion
      return !this.application.id || this.isReadOnly();
```

File: applications-frontend/src/components/Applicant.vue (hunk 1 above)
Comment (thomasmunduchira) [reply applications-frontend/src/components/Applicant.vue]: Done

File: applications-frontend/src/components/Applicant.vue (hunk 2 above)
Comment (thomasmunduchira) [reply applications-frontend/src/components/Applicant.vue]: Done

**Pull request merged**